torch/csrc/jit/symbolic_script.cpp

```diff
@@ -776,7 +776,7 @@ const std::vector<std::string> functions = {
             other_size = other.size()
             def backward(grad_output):
                 grad_self = (grad_output * condition.type_as(grad_output))._grad_sum_to_size(self_size)
-                grad_other = (grad_output * (1 - condition).type_as(grad_output))._grad_sum_to_size(other_size)
+                grad_other = (grad_output * (1 - condition.byte()).type_as(grad_output))._grad_sum_to_size(other_size)
```

Comment:
Shouldn't we just use logical negation here?

Reply:
are you tracking this somewhere in an issue?

Reply:
did you answer this?